scripts/README.md

```diff
@@ -7,18 +7,22 @@ Below is a list of scripts available, along with a simple description of what
 each one does. The details of what they are doing is available within the
 script.
 
-[`pre-bootstrap`](pre-bootstrap)
-Directs towards base development machine setup.
-
 [`bootstrap`](bootstrap)
 Installs project's direct dependencies e.g. npm packages.
 
+[`ci-deployment`](ci-deployment)
+Infrastructure related work.
+
+[`deploy`](deploy)
+Clone [ci-deployment](https://github.com/nhsuk/ci-deployment.git) repo and
+execute `deploy` script.
+
+[`pre-bootstrap`](pre-bootstrap)
```

Comment:
Why has pre-boostrap moved from before bootstrap? Makes sense to keep it in the running order.